metron-deployment/vagrant/Kerberos-setup.md

```diff
@@ -107,23 +107,23 @@ ${HDP_HOME}/kafka-broker/bin/kafka-topics.sh --zookeeper ${ZOOKEEPER}:2181 --cre
 12. Setup Kafka ACLs for the topics
   ```
 export KERB_USER=metron;
-for topic in bro enrichments indexing snort; do
+for topic in bro enrichments indexing snort yaf; do
```

Comment:
Yes, that was the reasoning.  I'm game with either way, I assumed that there a specific reason why yaf was used.  I'll update the instructions to consider limited resources.